### ISSUE TITLE
3-4 metacarpal synostosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7107]
**Preferred term label:**
3-4 metacarpal synostosis

**Synonyms**
Middle and ring finger metacarpal synostosis
Synostosis of third and the fourth metacarpal bones 

**Definition (free text, please give PubMed ID)**
A congenital hand malformation characterized by the coalescence of the coalescence of third and fourth metacarpals (i.e., those corresponding to the middle and ring fingers).
PMID:27698627
Comment:
Metacarpal synostosis is best described as a failure of differentiation causing a union between 2 adjacent metacarpals

**Parent term (use hpo.jax.org/app)**
Metacarpal synostosis HP:0009701

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
615994 

2/2   PMID:23692385

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:4000062
